cob_articulation_controller/ParamMsg[] params
duration target_duration
---
bool success
string error_message
